core/importer/partners_importer.py

```diff
@@ -28,6 +28,9 @@ def get_partner(elu_accession, name):
         try:
             partner = get_partner(elu_accession=partner_dict.get('external_id', None), 
                                   name=partner_dict.get('name'))
+            if self.skip_on_exist:
+                self.logger.warning(f'Partner with name \""{partner_dict.get("name")}"\" already found. It will be skipped.')
```

Comment:
would be more readable with:
`self.logger.warning(f'Partner with name "{partner_dict.get("name")}" already found. It will be skipped.')`

but I think it can be left as it is right now